src/modules/ExternalNotificationModule.cpp

```diff
@@ -128,6 +128,11 @@ int32_t ExternalNotificationModule::runOnce()
     }
 }
 
+bool ExternalNotificationModule::wantPacket(const meshtastic_MeshPacket *p)
+{
+    return MeshService::isTextPayload(p);
+}
+
 /**
```

Comment:
A similar change needs to go into the payload gatherer for the Store&Forward App.